plugins/parsers/registry.go

```diff
@@ -53,6 +54,9 @@ type Config struct {
 
 	// DefaultTags are the default tags that will be added to all parsed metrics.
 	DefaultTags map[string]string
+
+	// PromFormat only to Prometheus
+	PromFormat map[string]string
```

Comment:
These options in `Config` are more meant to be something that users would specify in the telegraf config file. From my understanding of the prometheus parser, PromFormat is something that gets specified by the headers in the http response?

ie, I don't think this needs to go here